### PR TITLE
61 hg20 model checks

### PR DESCRIPTION
**Pull request by kaustubanand**
Added caveats for low angle impacts and corrected the impact angle calculation.

### CODE REVIEW

File: src/collision/collision_regime.f90 [hunk 1]
```diff
@@ -364,7 +372,7 @@ function calc_theta(r_tar, v_tar, r_imp, v_imp) result(theta)
             x_cross_v(:) = distance(:) .cross. imp_vel(:) 
             sintheta = norm2(x_cross_v(:)) / norm2(distance(:)) / norm2(imp_vel(:))
 
-            theta = asin(sintheta) ! Find a more exact way to calculate theta
+            theta = (PI / 2) - abs(asin(sintheta)) ! Find a more exact way to calculate theta
```
Comment (profminton) [inline src/collision/collision_regime.f90:375]: What does the comment here mean? 
> Find a more exact way to calculate theta

**Review by profminton — CHANGES_REQUESTED** [review]
Clarify the comment

File: src/collision/collision_regime.f90 (hunk 1 above)
Comment (kaustubanand) [reply src/collision/collision_regime.f90]: Edited the comment to reflect the calculation. The angle is being calculated correctly with respect to the horizon.

File: src/collision/collision_regime.f90 [hunk 2]
```diff
@@ -364,7 +372,7 @@ function calc_theta(r_tar, v_tar, r_imp, v_imp) result(theta)
             x_cross_v(:) = distance(:) .cross. imp_vel(:)
```
Comment (profminton) [inline src/collision/collision_regime.f90:372]: Would this work?

```fortran
theta = PIBY2 - acos((distance(:) .dot. imp_vel(:)) / (.mag.distance(:) * .mag.imp_vel(:)))
```

**Review by profminton — APPROVED** [review]

**Pull request merged**